### PR TITLE
5 merchant invoices sorted

### LINKED ISSUE #5: Merchant Invoices Sorted
Merchant Dashboard Invoices sorted by least recent

As a merchant
When I visit my merchant dashboard
In the section for "Items Ready to Ship",
Next to each Item name I see the date that the invoice was created
And I see the date formatted like "Monday, July 18, 2019"
And I see that the list is ordered from oldest to newest

┆Issue is synchronized with this [Trello card](https://trello.com/c/ACQkBylF) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by Penitent0**
Completed User Story 5

Refactor merchant.rb items_not_shipped to be items_not_shipped_sorted_by_date

### CODE REVIEW

File: spec/features/merchants/dashboard_spec.rb [hunk 1]
```diff
@@ -118,21 +126,21 @@
       it 'Then I see link to my merchant items index (/merchants/merchant_id/items)' do
         visit "/merchants/#{@merchant_1.id}/dashboard"
 
-        expect(find_link('Items Index')[:href].should == "/merchants/#{@merchant_1.id}/items")
+        find_link({text: "Items Index", href: "/merchants/#{@merchant_1.id}/items"}).visible?
```
Comment (SandyyMarie) [inline spec/features/merchants/dashboard_spec.rb:129]: Thanks for fixing these!

File: spec/features/merchants/dashboard_spec.rb [hunk 2]
```diff
@@ -281,15 +289,69 @@
...
+          find_link({text: "#{@invoice_4.id}", href: "/merchants/#{@pretty_plumbing.id}/invoices/#{@invoice_4.id}"}).visible?
+        end
+
+        within("##{@toilet.id}") do
+          find_link({text: "#{@invoice_5.id}", href: "/merchants/#{@pretty_plumbing.id}/invoices/#{@invoice_5.id}"}).visible?
+        end
+      end
+    end
+
+    # As a merchant
```
Comment (SandyyMarie) [inline spec/features/merchants/dashboard_spec.rb:305]: We'll have to pick when we go back to refactor if we want to keep all the user stories tabbed in comments or clean them out for submission

**Review by SandyyMarie — APPROVED** [review]
Thanks Ken and Aleisha for knocking out the last kinks!

**Comment by SandyyMarie** [comment]
I'll review these merge conflicts in the morning since it only wants it to be done through command line and not GitHub

**Comment by Penitent0** [comment]
[Merchant Invoices Sorted](https://trello.com/c/ACQkBylF/36-merchant-invoices-sorted)

**Review by eriggs0207 — APPROVED** [review]

**Pull request merged**